### DISCUSSION TITLE
UI font is affected by osd-color since the latest release, how to make it white again?

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#14739] [Q&A]
Hi, until v.38 the UI overlay which is toggled with `I` wasn't affected by `osd-color` it only affected popups like subtitles on/off or chapter names. Now the overlay UI is the same like the popup messages. How to change that?

Thank you :)
The readability is not so good
![font](https://github.com/user-attachments/assets/66b4c54e-7593-416a-b2f5-175e3e7e8fe4)

**Comment by guidocella** [comment]
`--script-opts-append=stats-font_color=FFFFFF`

**Reply by geextahslex** [reply]
this doesn't work when I use `osd-color=0.6/0.3` for the "popup messages"
